### ISSUE TITLE
Config - array in array square brackets not supported.

### ISSUE THREAD

**Issue by veteran29** [SkaceKamen/vscode-sqflint#29]
It seems that in config files inner arrays can also use "[ ... ]" instead of "{ ... }".

This is valid and works in sqflint:
```sqf
class KPLIB_MapControl {
    stickX[] = {0.2, {"Gamma", 1, 1.5} };
};
```

This is valid and breaks the sqflint parser:
```sqf
class KPLIB_MapControl {
    stickX[] = {0.2, ["Gamma", 1, 1.5] };
};
```

This is not valid:
```sqf
class KPLIB_MapControl {
    stickX[] = [0.2, ["Gamma", 1, 1.5] ];
};
```

**Comment by SkaceKamen** [comment]
Heh, wouldn't think that's possible

**Comment by veteran29** [comment]
Yea me either. Arma is full of surprises ;)

**Comment by SkaceKamen** [comment]
Fixed in 0.9.1
